ui/src/components/home/EventsSectionCards.js

```diff
@@ -72,54 +71,39 @@ const EventsSectionCard = (props) => {
       }
     },
   };
-
   return (
     <>
       <Box borderRadius={10} w={width} h={height} mx={4}>
-        <VStack justifyContent="space-between">
-          <AspectRatio width="full" ratio={16 / 9}>
+        <VStack justifyContent="space-between" alignItems="flex-start">
+          <AspectRatio
+            width="90%"
+            ratio={16 / 9}
+            onClick={onOpen}
+            cursor="pointer"
+            zIndex={5}
+          >
             <Image borderRadius={10} objectFit="cover" src={event.imageUrl} />
           </AspectRatio>
-          <Heading maxWidth={["3xs","lg"]} fontSize={["sm", "3xl"]} color="black" isTruncated>
-            {event.title}
-          </Heading>
-          <HStack direction="space-between" spacing={[2, 6]}>
-            <Button
-              as={Link}
-              target="_blank"
-              href={event.signUpLink ? event.signUpLink : null}
-              isDisabled={event.signUpLink.length <= 0}
-              w={["8em", "10em"]}
-              fontSize={["0.8em", "1.2em"]}
-              size="sm"
-              variant="outline"
-              borderColor="#062A83"
-              color="#062A83"
-              _hover={{
-                borderColor: "#FFFFFF",
-                color: "#FFFFFF",
-                background: "#062A83",
-              }}
-            >
-              Sign up
-            </Button>
-            <Button
+          <VStack alignItems="flex-start">
+            <Heading
               onClick={onOpen}
-              w={["8em", "10em"]}
-              fontSize={["0.8em", "1.2em"]}
-              size="sm"
-              variant="outline"
-              borderColor="#062A83"
-              color="#062A83"
-              _hover={{
-                borderColor: "#FFFFFF",
-                color: "#FFFFFF",
-                background: "#062A83",
-              }}
+              marginBottom="5px"
+              maxWidth={['3xs', 'lg']}
+              fontSize={['sm', '2xl']}
+              color="#A5CBFF"
+              cursor="pointer"
+              isTruncated
             >
-              Learn more
-            </Button>
-          </HStack>
+              {event.title}
+            </Heading>
```

Comment:
[nitpick] title should be fontweight="semibold", could also add extra top padding or top margin. id suggest making the top and bottom spacing equal to look more like the design in figma. 
![Uploading Screenshot 2022-08-15 214834.png…]()